romancal/datamodels/schemas/reference_files/gain.schema.yaml

```diff
@@ -0,0 +1,14 @@
+%YAML 1.1
+---
+$schema: http://stsci.edu/schemas/asdf/asdf-schema-1.0.0
+id: "http://stsci.edu/schemas/roman_datamodel/reference_files/gain.schema"
+allOf:
+- $ref: referencefile.schema
+- $ref: keyword_gainfact.schema
+- type: object
+  properties:
+    data:
+      title: The detector gain map
+      default: 0.0
+      ndim: 2
+      datatype: float32
```

Comment:
If this is a data array why not use 

`$ref: ../asdf/core/ndarray-1.0.0`

@eslavich Is this the correct way to reference ndarray?

Reply:
I'd recommend this to save a redundant validation pass:

```yaml
tag: tag:stsci.edu:asdf/core/ndarray-1.0.0
```

There's an open issue to add this to the existing schemas: https://github.com/spacetelescope/romancal/issues/79

Reply:
Tagged.